platform/linux-generic/odp_packet_io.c

```diff
@@ -306,6 +329,7 @@ static odp_pktio_t setup_pktio_entry(const char *name, odp_pool_t pool,
 	snprintf(pktio_entry->s.full_name, sizeof(pktio_entry->s.full_name), "%s", name);
 	pktio_entry->s.pool = pool;
 	memcpy(&pktio_entry->s.param, param, sizeof(odp_pktio_param_t));
+	pktio_entry->s.tx_compl_pool = ODP_POOL_INVALID;
```

Comment:
This should be done now inside init_pktio_entry(), since the value does not require anything from the parameters.